usd/src/usd_parser/usd_model/WorldInterface.hh

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#ifndef USD_PARSER_USD_MODEL_WORLD_INTERFACE_HH_
+#define USD_PARSER_USD_MODEL_WORLD_INTERFACE_HH_
+
+#include <ostream>
+#include <string>
+#include <vector>
+
+#include <ignition/math/Vector3.hh>
+
+#include <sdf/Light.hh>
+#include <sdf/sdf_config.h>
+
+namespace sdf
+{
+  // Inline bracke to help doxygen filtering.
+  inline namespace SDF_VERSION_NAMESPACE {
+  //
+  namespace usd
+  {
+    /// \brief This class store data about the world
+    class WorldInterface {
```

Comment:
Is there a particular reason why the file that defines this class is in a `usd_model` subdirectory? What if we put this file in the `usd/src/usd_parser/` directory with the other source files?

Reply:
I just want to differentiate that these classes will create a model of the USD, they are not part of the parser.